resources/assetstore/charts/minio/templates/content-upload-configmap.yaml

```diff
@@ -13,6 +13,6 @@ data:
   APP_UPLOAD_ENDPOINT_WITH_PORT: '{{ template "minio.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.service.port }}'
   APP_BUCKET_NAME: content
   APP_BUCKET_REGION: us-east-1
-  APP_EXTERNAL_ENDPOINT: https://minio.{{ .Values.global.ingress.domainName }}
+  APP_EXTERNAL_ENDPOINT: {{ if .Values.global.minioExternalEndpoint }}{{ .Values.global.minioExternalEndpoint }}{{ else }}'https://minio.{{ .Values.global.ingress.domainName }}'{{ end }}
```

Comment:
If you use global variables in this condition and in various places in asset-store charts, maybe would be better to extract this logic to global values as storeExternalEndpoint?

Reply:
`values.yaml` may contain only values, logic is not supported.